src/Particle/Validator/Rule/Between.php

```diff
@@ -28,7 +33,8 @@ class Between extends Rule
      * @var array
      */
     protected $messageTemplates = [
-        self::NOT_BETWEEN => '{{ name }} must be between {{ min }} and {{ max }}'
+        self::TOO_BIG => '{{ name }} is too big, upper limit is {{ max }}',
+        self::TOO_SMALL => '{{ name }} is too small, lower limit is {{ min }}',
```

Comment:
I think `{{ name }} must be greater than {{ min }}` is even more descriptive. "is too low and must be higher than" is duplication ;)